<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>detectDialog</class>
 <widget class="QMainWindow" name="detectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1106</width>
    <height>446</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>910</width>
    <height>337</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Human Detection</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QWidget" name="widget" native="true">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0" colspan="2">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="font">
              <font>
               <family>MS Shell Dlg 2</family>
               <pointsize>16</pointsize>
               <weight>9</weight>
               <italic>false</italic>
               <bold>false</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">font: 75 16pt &quot;MS Shell Dlg 2&quot;;</string>
             </property>
             <property name="text">
              <string>SystemWide Human Prosence Detection</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>30</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>0</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>MS Shell Dlg 2</family>
               <pointsize>14</pointsize>
               <weight>9</weight>
               <italic>false</italic>
               <bold>false</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;
color: rgb(85, 155, 0);</string>
             </property>
             <property name="text">
              <string>Detected :  1+ Human Presence</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="font">
              <font>
               <family>MS Shell Dlg 2</family>
               <pointsize>14</pointsize>
               <weight>9</weight>
               <italic>false</italic>
               <bold>false</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;
color: rgb(185, 89, 0);</string>
             </property>
             <property name="text">
              <string>Refresh Rate : 50Hz</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>8</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QTableWidget" name="tableWidget">
           <property name="styleSheet">
            <string notr="true">background-color:#e6e6e6;</string>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <attribute name="horizontalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="horizontalHeaderCascadingSectionResizes">
            <bool>false</bool>
           </attribute>
           <attribute name="horizontalHeaderHighlightSections">
            <bool>true</bool>
           </attribute>
           <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
            <bool>false</bool>
           </attribute>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <row>
            <property name="text">
             <string>New Row</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>Human Presence Detection</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>Locomotion</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>Movement</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>Stationary Presence(#breaths/min)</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>Number of Human Presence</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>  -</string>
            </property>
           </row>
           <column>
            <property name="text">
             <string>Locations</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Node1</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Node2</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Node3</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Node4</string>
            </property>
           </column>
           <item row="0" column="0">
            <property name="text">
             <string>Locations</string>
            </property>
           </item>
           <item row="0" column="1">
            <property name="text">
             <string>Node1</string>
            </property>
           </item>
           <item row="0" column="2">
            <property name="text">
             <string>Node2</string>
            </property>
           </item>
           <item row="0" column="3">
            <property name="text">
             <string>Node3</string>
            </property>
           </item>
           <item row="0" column="4">
            <property name="text">
             <string>Node4</string>
            </property>
           </item>
           <item row="1" column="0">
            <property name="text">
             <string>Human Presence Detection</string>
            </property>
           </item>
           <item row="1" column="1">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="1" column="2">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="1" column="3">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="1" column="4">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="2" column="0">
            <property name="text">
             <string>Locomotion</string>
            </property>
           </item>
           <item row="2" column="1">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="2" column="2">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="2" column="3">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="2" column="4">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="3" column="0">
            <property name="text">
             <string>Movement</string>
            </property>
           </item>
           <item row="3" column="1">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="3" column="2">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="3" column="3">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="3" column="4">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="4" column="0">
            <property name="text">
             <string>Stationary Presence(#breaths/min)</string>
            </property>
           </item>
           <item row="4" column="1">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="4" column="2">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="4" column="3">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="4" column="4">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="5" column="0">
            <property name="text">
             <string>Number of Human Presence</string>
            </property>
           </item>
           <item row="5" column="1">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="5" column="2">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="5" column="3">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="5" column="4">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="6" column="0">
            <property name="text">
             <string> -</string>
            </property>
           </item>
           <item row="6" column="1">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="6" column="2">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="6" column="3">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="6" column="4">
            <property name="text">
             <string/>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="ipaddressLabel">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>200</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>230</y>
      <width>180</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>180</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>200</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1106</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
